grpclb/src/main/java/io/grpc/grpclb/GrpclbLoadBalancer.java

```diff
@@ -78,6 +83,20 @@
   private static final Logger logger = Logger.getLogger(GrpclbLoadBalancer.class.getName());
 
   @VisibleForTesting
+  static final Map<DropType, Status> DROP_STATUSES;
+
+  static {
+    HashMap<DropType, Status> map = new HashMap<DropType, Status>();
```

Comment:
* you can use `EnumMap` at the beginning.
`EnumMap<DropType, PickResult> map = new EnumMap<DropType, PickResult>(DropType.class);`
* you can use `PickResult` instead of `Status` (also rename `DROP_STATUSES` to `DROP_RESULTS`), then you don't have to create a new `PickResult` instance later each time. This can produce less garbage.